js/stdlib/ts/ALGO.ts

```diff
@@ -211,11 +211,18 @@ type AppSchema = {
 }
 type AccountInfo = {
   'amount': bigint,
-  'assets'?: Array<AccountAssetInfo>,
+  'assets'?: Array<AssetHolding>,
   'apps-local-state'?: Array<AppState>,
   'apps-total-schema'?: AppSchema,
   'created-apps'?: Array<AppInfo>
 };
+type AccountAssetInfo = {
+  'asset-holding'?: AssetHolding
+};
+type AccountApplicationInfo = {
```

Comment:
The naming scheme would be to have this be `Res` thing, because the node returns things tagged with the `round` and then have another type that is used for both the indexer and the node query result